src/device_info.sh

```diff
@@ -73,13 +86,17 @@ function get_ram()
   case "$target" in
     1) # VM_TARGET
       ram="$(printf '%s\n' "${vm_config[qemu_hw_options]}" | sed --regexp-extended 's/.*-m ?([0-9]+).*/\1/')"
-      ram="$(numfmt --from-unit=M --to-unit=K "$ram")"
+      cmd="numfmt --from-unit=M --to-unit=K ${ram}"
+      show_verbose "$flag" "$cmd"
+      ram=$(cmd_manager 'SILENT' "$cmd")
       ;;
     2) # LOCAL_TARGET
-      ram=$(cmd_manager "$flag" "$cmd")
+      show_verbose "$flag" "$cmd"
+      ram=$(cmd_manager 'SILENT' "$cmd")
```

Comment:
Just to be sure, you are using `show_verbose "$flag" "$cmd"`, because `ram=$(cmd_manager "$flag" "$cmd")` (with `flag=VERBOSE`) outputs the command to the `ram` variable instead of to the standard output, right? I tested without `show_verbose` and it seems that's the case.

Reply:
Yeah, since we need to capture the output of the command, using the VERBOSE option will mess with the collected value. I added the `SILENT` flag and created the `show_verbose` option to workaround this problem. Maybe in the future, we can find a clever way to improve `cmd_manager` so that the VERBOSE flag is not captured in these cases. For now, the `show_verbose` is the easy solution.